.travis.yml

```diff
@@ -0,0 +1,60 @@
+dist: xenial
+sudo: required
+
+stages:
+  - Build
+  - Tests
+
+cache:
+  timeout: 1000
+  directories:
+    - $CACHE_DIR
+
+env:
+  global:
+    - CACHE_DIR=$HOME/.cache/docker
+    - CACHE_FILE_IMAGE=$CACHE_DIR/php-fpm.tar.gz
+    - ES_VERSION=6.3.2
```

Comment:
maybe we could put env variable for versions of services like ELASTICSEARCH, REDIS, POSTGRES into travis UI like we have it with COMPOSER_MEMORY_LIMIT